web/src/components/CardForm.test.js

```diff
@@ -11,10 +11,10 @@ describe('card form wrapper', () => {
     history.goBack.resetHistory();
   });
 
-  test('renders without an id on the container if id prop is false', () => {
+  test('renders without an id on the container if id prop is null', () => {
     expect(
       shallow(
-        <CardForm id={false} title="test" history={history}>
+        <CardForm id={null} title="test" history={history}>
```

Comment:
Looking at the codebase, we never set the id when using the `CardForm` component. Seems like an unnecessary test.

Reply:
Reading a little further into the codebase, looks like we're setting this to `'start-main-content'` when not specified. I'll leave this be.